cloudinit/sources/DataSourceRbxCloud.py

```diff
@@ -153,6 +153,7 @@ def read_user_data_callback(mount_dir):
 
     data = {
         'userdata': user_data,
+        'vendordata': meta_data,
```

Comment:
Added to expose vendordata to fullfil datasource API.

Reply:
vendordata is cloud-provided cloud-config (per https://cloudinit.readthedocs.io/en/latest/topics/vendordata.html). Looking at the way that `meta_data` is used in this function, it looks like it's a dictionary loaded from JSON, which isn't the correct format for vendordata.

Am I missing something?